src/components/App.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState, useEffect } from "react";
+import "../styles/App.css";
+import getForecast from "../requests/getForecast";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import SearchForm from "./SearchForm";
+
+function App() {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({
+    city: "Hi",
+    country: "want to search the weather?",
```

Comment:
This is the same for city. It would be better to either initialise it with an actual country and city as a default, or use empty strings and render something different if you have no city and no country